webapp/layouts/error.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <div class="error-container">
+    <base-icon class="error-icon" name="exclamation-circle" />
+    <br />
+    <span v-if="error.message === 'This page could not be found'" class="error-message">
```

Comment:
I was not able to locate the place where the default 404 error message comes from. So I catch it here. Any ideas?

Reply:
You could dispatch the error code:

```js
computed: {
  message() {
    const mapping = {
       403: 'error-pages.not-authorized',
       404: // ...
       500: // ...
    }
    const key = mapping(this.error.code) || 'error.default'
    return this.$(key)
  }
}